nixos/modules/services/x11/desktop-managers/cinnamon.nix

```diff
@@ -123,14 +123,19 @@ in
         cinnamon-screensaver = {};
       };
 
-      environment.systemPackages = with pkgs.cinnamon // pkgs; [
+      environment.systemPackages = with pkgs.cinnamon // pkgs; ([
+        # session requirements
+        cinnamon-common
+        cinnamon-screensaver
+        nemo
+        networkmanagerapplet # also nm-applet not needed
+      ] ++ utils.removePackagesByName [
```

Comment:
not sure if removing some of these packages will accidently break the session. session requirements simply meant what the session desktop file lists as a requirement.

Reply:
TBH I don't use excludePackages myself so :woman_shrugging: , but anyway now the list only includes accessibility package and theme package, hope it will make more sense